apps/prairielearn/elements/pl-number-input/pl-number-input.py

```diff
@@ -217,55 +240,42 @@ def render(element_html: str, data: pl.QuestionData) -> str:
         info_params["allow_complex"] = pl.get_boolean_attrib(
             element, "allow-complex", ALLOW_COMPLEX_DEFAULT
         )
-        info_params["show_info"] = pl.get_boolean_attrib(
-            element, "show-help-text", SHOW_HELP_TEXT_DEFAULT
-        )
+        info_params["show_info"] = show_help_text
         info_params["allow_fractions"] = allow_fractions
 
         # Find the true answer to be able to display it in the info popup
         ans_true = None
-        if pl.get_boolean_attrib(
-            element, "show-correct-answer", SHOW_CORRECT_ANSWER_DEFAULT
-        ):
+        if show_correct_answer:
             ans_true = format_true_ans(element, data, name)
```

Comment:
Same weird logic as in previous commit.

Reply:
Same as [above](https://github.com/PrairieLearn/PrairieLearn/pull/8435#discussion_r1385799393).